commands/project.go

```diff
@@ -0,0 +1,93 @@
+package commands
+
+//handle the project logic for rhmap.
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/feedhenry/rhm/storage"
+	"github.com/urfave/cli"
+)
+
+//ProjectCmd constructs the required writer in order to send the response to the right place.
+type projectCmd struct {
+	out      io.Writer
+	in       io.Reader
+	response func(*http.Request) (*http.Response, error)
+	//response   http.Client
+	newrequest func(string, string, io.Reader) (*http.Request, error)
```

Comment:
Not sure we need this as an explicit dependency. The reasoning here is that when testing we want to mock the response from the server, but the building of the request object is not important to mock out.